ajbleakley/register-app
# Feature - Install Mezzio framework

### What is the (feature|problem)?

Relates to: [AS A software engineer I WANT TO use a framework SO THAT I can spend less time on boilerplate and focus on business logic](https://github.com/users/ajbleakley/projects/1/views/1?pane=issue&itemId=65622844)

### What is the solution?

In my proposal I opted to use a lightweight, non-opinionated PHP framework called [Mezzio](https://docs.mezzio.dev/mezzio/). To install it I followed the [official getting started instructions](https://docs.mezzio.dev/mezzio/v3/getting-started/quick-start/).

### What areas of the app does it impact?

No customer facing impact. Makes life easier for developers as they can spend less time writing boilerplate code.

### How to test?

For now you can spin up the application using PHP's built-in web server:
1. Clone repository.
2. Run `composer install`
3. Run `composer run --timeout=0 serve` to serve the application.
4. Visit http://0.0.0.0:8080/api/ping in your web browser to verify the application is running.
